doc/odoc-config-eio.sexp

```diff
@@ -0,0 +1,3 @@
+(libraries unix)
+(packages eio_main bigstring)
```

Comment:
This adds the unix library to all libraries build as part of the eio package. This was required because the [eio] library was referring to `Unix.SO_REUSEADDR` in net.mli